CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
   staging server instead of the live server when `--dry-run` is used.
 * Updated certbot-dns-google to depend on newer versions of
   google-api-python-client and oauth2client.
+* Deprecated config_changes
```

Comment:
nit: We've used "deprecate" to refer to features that still exist but will be removed in the future.

I'd essentially copy the changelog entries from https://github.com/certbot/certbot/pull/7198 but replace "has been deprecated and will be removed..." to "has been removed."